Source/cmake/OptionsWin.cmake

```diff
@@ -84,6 +84,7 @@ find_package(PNG 1.6.34 REQUIRED)
 find_package(SQLite3 3.23.1 REQUIRED)
 find_package(ZLIB 1.2.11 REQUIRED)
 find_package(LibPSL 0.20.2 REQUIRED)
+find_package(AVIF 0.9.0 REQUIRED)
```

Comment:
Why don't we move this into the Optional packages section and if its found then set `USE_AVIF`.

Reply:
WebP, which is major than AVIF, is also optional.
 
https://github.com/WebKit/WebKit/blob/45e64ff33376b8df1116d92a6abe15ede376731e/Source/cmake/OptionsWin.cmake#L106-L109